### ISSUE TITLE
[css-values] unify CSS wide keywords and defaulting keywords

### ISSUE THREAD

**Issue by yisibl** [w3c/csswg-drafts#7170]
These keyword definitions are currently scattered throughout the specification, making it easy for other CSS parsers or engine implementers to miss them. e.g.

- esbuild: https://github.com/sapphi-red/esbuild/blob/255a22fe22a98133da03c3a5df80238c91ad41b0/internal/css_parser/css_decls_font_family.go#L11-L15

```go
var wideKeywords = map[string]bool{
	"initial": true,
	"inherit": true,
	"unset":   true,
}
```

- SWC https://github.com/swc-project/swc/blob/9d6a57c6e5954a6fcd35a55a27bf7067eec5fd9b/crates/swc_css_parser/src/parser/value/mod.rs#L1420-L1425

```rust
match &*value.to_ascii_lowercase() {
    "initial" | "inherit" | "unset" | "revert" | "default" => {
        return Err(Error::new(span, ErrorKind::InvalidCustomIdent(raw)));
    }
    _ => {}
}
```

- cssnano https://github.com/cssnano/cssnano/blob/2901a9c88ed5ac47c612e1b924669f28ae920347/packages/postcss-minify-font-values/src/lib/minify-family.js#L17

```js
const globalKeywords = ['inherit', 'initial', 'unset'];
```

## css-values-4
https://www.w3.org/TR/css-values-4/#common-keywords

Defined:

- `initial`
- `inherit`
- `unset`

## css-cascade-4

https://drafts.csswg.org/css-cascade/#defaulting-keywords

Defined: 

- `initial`
- `inherit`
- `unset`
- **`revert`**

## css-cascade-5

https://drafts.csswg.org/css-cascade-5/#revert-layer

Defined:

- `initial`
- `inherit`
- `unset`
- `revert`
- **`revert-layer`**

## css-values-4

https://www.w3.org/TR/css-values-4/#ref-for-css-wide-keywords

> The CSS-wide keywords are not valid `<custom-ident>`s. The `default` keyword is reserved and is also not a valid `<custom-ident>`

There is a `default` keyword.

**Comment by tabatkins** [comment]
The definitions are already unified in the Cascade specs (and later levels replace preceding levels, so Cascade 5 is the canonical definition). Values merely mentions them, and points to Cascade for their definition; Fonts mentions them in an example (including the reserved "default" keyword)

**Comment by yisibl** [comment]
@tabatkins Thank you for your reply.

Yes, the CSS fonts spec mentions the terms `common-keywords|CSS-wide keyword`, so can we change them all to `CSS-wide keyword` in Cascade 5? Too many terms can be confusing and detrimental to communication.

> Any identifier which could be misinterpreted as a pre-defined keyword in the font-family value definition, or the common-keywords|CSS-wide keyword values, is not allowed.


Also is it possible to add the "default" keyword to the [`Explicit Defaulting`](https://drafts.csswg.org/css-cascade-5/#defaulting-keywords) section in Cascade 5?

**Comment by fantasai** [comment]
Hi @yisibl, we've refactored the spec text around CSS-wide keywords in https://github.com/w3c/csswg-drafts/commit/9b95cab15a325abb5934590e85c830811ba1ba70

For “default”, this is merely excluded from the `<custom-ident>` production, since it is not a valid CSS-wide keyword anywhere. We are also discussing adding additional keywords to the `<custom-ident>` exclusion list in #7431.

Closing out this issue, but let us know if there are any remaining problem.
